apps/web/src/pages/integrations/components/multi-provider/UpdateProviderSidebar.tsx

```diff
@@ -159,23 +157,38 @@ export function UpdateProviderSidebar({
     EmailProviderIdEnum.Novu === selectedProvider?.providerId
   ) {
     return (
-      <Sidebar
-        isOpened={isSidebarOpened}
-        isLoading={areProvidersLoading || areEnvironmentsLoading}
-        onClose={onSidebarClose}
-        customHeader={
-          <Group spacing={12}>
-            <ProviderImage providerId={selectedProvider?.providerId} />
-            <Title size={2}>{selectedProvider?.displayName ?? ''}</Title>
-            <Free>🎉 Free</Free>
-          </Group>
-        }
-        data-test-id="update-provider-sidebar-novu"
-      >
-        <ProviderInfo provider={selectedProvider} environments={environments} />
-
-        <NovuProviderSidebarContent provider={selectedProvider} />
-      </Sidebar>
+      <FormProvider {...methods}>
```

Comment:
I keept adding stuff up here because I did not like to make the part lower down even more complex.